### DISCUSSION TITLE
AI for generating landscapes of planets

### DISCUSSION THREAD

**Discussion by andreadev-it** [endless-sky/endless-sky#8787] [General]
Hi everyone,
I was wondering: how are AI generated images viewed in the community? I think it has huge potential for planet landscapes and similar other pictures. I recently had access to Leonardo.ai and tried imputting a prompt about a futuristic landscape of a planet. This is what came out, I think it's pretty good:
![Leonardo_Signature_Landscape_of_a_technological_advanced_plane_0](https://github.com/endless-sky/endless-sky/assets/68549121/fd0796c7-5e4b-4a01-8a64-d77a38de3257)

I tried looking for similar topics in the discussions, but couldn't find anything

**Comment by bene-dictator** [comment]
There is discussion on the community discord server about it on `#artwork-creating`. We don't use any at the moment, and if you ask me, although the image looks great it has too many pastel colours compared to the photos we use for landscapes. Of course it could change but it would look jarring more than anything.

The only time we use any computer-generated landscape images are some human space station interiors, I think. And they do stand out weirdly compared to other photos.

**Comment by Saugia** [comment]
There was a decently long conversation, but the TLDR is we do not accept any sort of AI-made content, especially for artwork. Several of our creators would rather put forth the time and effort to make their own land scenes instead of letting them be randomly generated, and the copyright issue behind AI art is too messy for us to consider using it.

**Reply by andreadev-it** [reply]
Ok, thanks to both of you for the heads up, not being on discord I couldn't see the conversation. 
For the quality, I think it could easily be improved by better prompting and post production work, but I can see how the copyright issues can be uneasing.
At least now we have a trace about this on github too 👍

**Comment by zitraksMD** [comment]
Hey, really cool idea with AI for generating landscapes of planets, i think it was already released in new AI from Adobe, when it generates what we ask for in a picture, I think we can work something out with that! Yes and take photos of land landscapes I think we can with the [Free AI Photo Generator](https://ai4free.ai/free_ai_photo_generator/) as well. Now in our world there are no impossible tasks that AI can't cope with, because this field is developing at an insane rate!

**Comment by Saugia** [comment]
As I have said above, we are not using AI to generate imagery for Endless Sky due to both copyright issues and strong desire from the community to create the work themselves rather than let something else do it for them.

**Comment by Zitchas** [comment]
As Saugia said, the current community policy is to not include AI generated imagery in the vanilla game. This policy would likewise apply to the official high-dpi plugin.

That being said, as a reminder, plug-ins are independent objects, and thus are free to set their own copyright and content policies. As such, if you want to make a plug-in adding alien worlds using AI generated imagery, then you are free to do so. You could even replace some of the existing imagery to make alien worlds look more alien. I suspect that there are more than a few people who would download and use such a plug-in.

Setting aside the policy side of things, the image looks quite interesting. I'd love to see some landscapes that are actually not found on Earth - True *alien* landscapes. I love exploration, and after a while it gets boring to go to a new planet in an exotic local and... see another view of somewhere else on Earth.
